extensions/wikia/Email/EmailController.class.php

```diff
@@ -547,10 +543,6 @@ public function assertValidUser( $user ) {
 		if ( !$user instanceof \User ) {
 			throw new Fatal( 'Unable to create user object');
 		}
-
-		if ( $user->getId() == 0 ) {
-			throw new Fatal( 'Unable to find user' );
-		}
```

Comment:
This is the main fix for this ticket.  The other stuff is just cleaning up the error reporting.